stable/wordpress/templates/deployment.yaml

```diff
@@ -26,11 +26,7 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
         env:
         - name: ALLOW_EMPTY_PASSWORD
-        {{- if .Values.allowEmptyPassword }}
-          value: "yes"
-        {{- else }}
-          value: "no"
-        {{- end }}
+          value: {{ .Values.allowEmptyPassword }}
```

Comment:
You did the opposite. I'd rather leave it as:

```
value: {{ .Values.allowEmptyPassword | quote }}
```

And correct values.yaml so it looks like:

```
...
allowEmptyPassword: yes
...
```